app/helpers/application_helper.rb

```diff
@@ -16,6 +16,13 @@ def tooltip(text)
     )
   end
 
+  def alert_category(key)
+    key = key.to_s.downcase
+    return 'oops' if key.in? %w(error alert)
```

Comment:
this is interesting / cool -- would it make sense to update the controllers to use one type of alert or the other rather than looking for two types and giving them the same style?